orchestration-shared/src/main/java/uk/gov/di/orchestration/shared/services/LogoutService.java

```diff
@@ -153,7 +162,8 @@ public APIGatewayProxyResponseEvent generateLogoutResponse(
                         .withIpAddress(extractIpAddress(input))
                         .withPersistentSessionId(
                                 extractPersistentIdFromCookieHeader(input.getHeaders()))
-                        .withSessionId(sessionId.orElse(null));
+                        .withSessionId(sessionId.orElse(null))
```

Comment:
Would it make sense to build the audit user in the handler and pass that through, rather than the individual fields?

Reply:
It's possible, and it would make things neater, but also means that the audit user is quite separated from where it's actually used. I'm not sure if it's that valuable right now. I'd like to spend some time refactoring LogoutHandler / LogoutService after this ticket